hoomd/md/integrate.py

```diff
@@ -169,14 +169,18 @@ class Integrator(_DynamicIntegrator):
             rigid bodies in the simulation.
 
 
-    The following classes can be used as elements in `methods`
+    Classes of the following modules can be used as elements in `methods`:
 
     - `hoomd.md.methods.Brownian`
     - `hoomd.md.methods.Langevin`
     - `hoomd.md.methods.NVE`
     - `hoomd.md.methods.NVT`
     - `hoomd.md.methods.NPT`
 
+    - `hoomd.md.methods.rattle.Brownian`
+    - `hoomd.md.methods.rattle.Langevin`
+    - `hoomd.md.methods.rattle.NVE`
+
```

Comment:
```suggestion
    Classes of the following modules can be used as elements in `methods`:

    - `hoomd.md.methods`
    - `hoomd.md.methods.rattle`
```
The updated description mentions modules, so list the modules.